manifests/canary-checker.yaml

```diff
@@ -228,13 +242,18 @@ spec:
   jobLabel: canary-checker
   selector:
     matchLabels:
+      app.kubernetes.io/instance: canary-checker
+      app.kubernetes.io/name: canary-checker
       control-plane: canary-checker
 ---
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   annotations:
-    kubernetes.io/tls-acme: "true"
```

Comment:
this needs to stay true?  OR are we currently patching it in later along with oauth instructions?

Reply:
Fixed